packages/suite/src/views/wallet/coinmarket/savings/bank-account/index.tsx

```diff
@@ -0,0 +1,124 @@
+        onSubmit,
+        handleSubmit,
+        formState,
+        isWatchingKYCStatus,
+        isSavingsTradeLoading,
+        control,
+    } = useCoinmarketSavingsBankAccount(props);
+
+    if (isSavingsTradeLoading) {
+        return <Translation id="TR_LOADING" />;
+    }
+    const typeSelectName = 'typeOption';
+    const nameInputName = 'name';
+    const routingNumberInputName = 'routingNumber';
+    const accountNumberInputName = 'accountNumber';
+    const bankAccountOwnerInputName = 'bankAccountOwner';
+    const { isSubmitting } = formState;
+    const { name, routingNumber, accountNumber, bankAccountOwner } = getValues();
+    const canSubmit = Object.keys(errors).length === 0 && !isSubmitting;
+    return (
+        <>
+            {isWatchingKYCStatus && <KYCInProgress />}
+            <Header>
+                <Translation id="TR_SAVINGS_BANK_ACCOUNT_HEADER" />
+            </Header>
+            <form onSubmit={handleSubmit(onSubmit)}>
+                <Controller
+                    control={control}
+                    name={typeSelectName}
+                    defaultValue={buildUsBankAccountType(UsBankAccountTypes[0])}
+                    render={({ onChange, value }) => (
+                        <>
+                            <Select
+                                options={UsBankAccountTypes.map(buildUsBankAccountType)}
+                                value={value}
+                                isClearable={false}
+                                hideTextCursor
+                                label={<Translation id="TR_SAVINGS_BANK_ACCOUNT_TYPE_LABEL" />}
+                                onChange={(selected: any) => {
+                                    onChange(selected);
+                                }}
+                            />
+                            <BankAccountTypeNote>
+                                {value.value === UsBankAccountTypes[1] && (
+                                    <Translation id="TR_SAVINGS_BANK_ACCOUNT_TYPE_NOTE" />
+                                )}
+                            </BankAccountTypeNote>
+                        </>
+                    )}
+                />
+                <Input
+                    placeholder="Checking account"
+                    label={<Translation id="TR_SAVINGS_BANK_ACCOUNT_NAME_LABEL" />}
+                    name={nameInputName}
+                    innerRef={register()}
+                    inputState={getInputState(errors.name, name)}
+                    bottomText={<InputError error={errors[nameInputName]} />}
+                />
+                <Input
+                    label={<Translation id="TR_SAVINGS_BANK_ACCOUNT_ROUTING_NUMBER_LABEL" />}
+                    name={routingNumberInputName}
+                    inputState={getInputState(errors.routingNumber, routingNumber)}
+                    innerRef={register({
+                        required: 'TR_SAVINGS_BANK_ACCOUNT_ROUTING_NUMBER_REQUIRED',
```

Comment:
Please, use the `validation` package to create the schema, you can find the implementation example here 😎 https://github.com/trezor/trezor-suite/blob/23a8ae6a09e06a971e681d5bbfc9227d11a79540/packages/suite/src/hooks/wallet/sign-verify/useSignVerifyForm.ts#L35